src/viur/core/modules/file.py

```diff
@@ -352,6 +352,14 @@ class FileLeafSkel(TreeSkel):
         readOnly=True,
         visible=False,
     )
+    crc32c_hash = StringBone(
+        descr="CRC32C hash for this File",
+        readOnly=True
+    )
+    md5_hash = StringBone(
+        descr="MD5 hash for this File",
+        readOnly=True
+    )
```

Comment:
Can we rename it to checksum to clarify that this feature is used to identify same files?
```suggestion
    crc32c_checksum = StringBone(
        descr="CRC32C checksum",
        readOnly=True,
    )
    md5_checksum = StringBone(
        descr="MD5 checksum",
        readOnly=True,
    )
```